src/pages/CreateAccount.js

```diff
@@ -130,6 +155,7 @@ export default function CreateAccount({ api, handleSuccessfulSignIn }) {
                     className="bold underlined-link"
                     href="https://raw.githubusercontent.com/zeeguu/browser-extension/main/PRIVACY.md"
                     target={isMobile() ? "_self" : "_blank"}
+                    rel="noreferrer"
```

Comment:
Should we make this into a Modal? We can fetch the text from the URL and display it through the modal which might be better to do than using a raw text file?

![image](https://github.com/zeeguu/web/assets/17390076/dc8534e9-5ae0-48ad-810e-30f41ac54445)

Reply:
The privacy policy will have its subpage on the homepage. But at the moment, when I see it now as a modal - I agree - it looks better. However, when working on it - I was asked by @mircealungu not to turn it into a modal but to provide a link to the text instead. I can still modify it as suggested if we all agree on this change :)

Reply:
I said to leave it on GH as it's the fastest way. If Tiago has implemented already a nicer version, then he should send a PR to your PR and we should include it?